File: 1/1.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    length=nums.length;
    for (let i = 0; i < length; i++) {
        for (let j = i+1; j < length; j++) {
            if (nums[i]+nums[j]==target){
                return [i,j];
            }
        }
    }
};

nums = [2,7,11,15];
target = 9;
console.log(twoSum(nums, target)); 

File: 1/1.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        length = len(nums)
        for i in range(0, length):
            for j in range(i + 1, length):
                if (nums[j] == target - nums[i]):
                    return [i, j]


if __name__ == "__main__":
    nums = [3, 2, 4]
    target = 6
    s = Solution()
    print(s.twoSum(nums, target))
